evap/development/fixtures/test_data.json

```diff
@@ -130235,6 +130235,69 @@
     "cc_users": []
   }
 },
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "oma.abner@institution.duplicate.com",
+    "title": null,
+    "first_name": "Oma",
+    "last_name": "Abner",
+    "language": null,
+    "is_proxy_user": false,
+    "login_key": null,
+    "login_key_valid_until": null,
+    "is_active": true,
+    "groups": [],
+    "user_permissions": [],
+    "delegates": [],
+    "cc_users": []
+  }
+},
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "lulu.ackerman@institution.duplicate.com",
+    "title": null,
+    "first_name": "Lulu",
+    "last_name": "Ackermann",
+    "language": null,
+    "is_proxy_user": false,
+    "login_key": null,
+    "login_key_valid_until": null,
+    "is_active": true,
+    "groups": [],
+    "user_permissions": [],
+    "delegates": [],
+    "cc_users": []
+  }
+},
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "echo.andre@institution.duplicate.com",
+    "title": null,
+    "first_name": "Echo",
+    "last_name": "Andre",
+    "language": null,
+    "is_proxy_user": false,
+    "login_key": null,
+    "login_key_valid_until": null,
+    "is_active": true,
+    "groups": [],
+    "user_permissions": [],
+    "delegates": [],
+    "cc_users": []
+  }
+},
```

Comment:
The idea here probably is to have some content on that page when loading it? I think for that, a single new user suffices. Could you undo all other changes to this file? (We can help you with that at a meeting)